src/robusta/integrations/kubernetes/base_event.py

```
@@ -31,3 +31,39 @@ def create_default_finding(self) -> Finding:
+            name = 'Unresolved'
+            namespace = None
+            labels = {}
+            annotations = {}
+        
+        # Get the kind from the object
+        kind = getattr(self.obj, 'kind', None) or 'Unknown'
+        
+        # Determine the subject type from the kind
+        try:
+            subject_type = FindingSubjectType.from_kind(kind)
+        except:
```

Flake8 (7.3.0)

[error] do not use bare 'except'

(E722)